src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -356,4 +375,10 @@ export class SkyListViewGridComponent
     }
     return true;
   }
+
+  private arraysEqual(oldColumns: any[], newColumns: any[]) {
```

Comment:
Nice private function here! Should these variables be named `columns` since this could compare any array? Maybe just `arrayA` and `arrayB`?

Reply:
Done